entities/library.rb

```diff
@@ -0,0 +1,27 @@
+class Library
+  include Statistics
+  include Validator
+  include DB
+
+  attr_reader :authors, :books, :readers, :orders
+
+  def initialize
+    @authors = []
+    @books   = []
+    @readers = []
+    @orders  = []
+    entities_hash = { authors: @authors, books: @books, readers: @readers, orders: @orders }
+    DB.load(entities_hash)
```

Comment:
and where does the loaded entities move?
they should be stored to @authors, @books ....